kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -254,57 +255,40 @@
         deleteSection,
       } = injectQuizCreation();
 
-      const { panelClosing } = toRefs(props);
-
       const showConfirmationModal = ref(false);
 
       function handleCancelClose() {
         showConfirmationModal.value = false;
-        context.emit('cancelClosePanel');
       }
 
       function handleConfirmClose() {
-        handleCancelClose();
         context.emit('closePanel');
       }
 
-      const selectedQuestionOrder = ref(activeSection.value.learners_see_fixed_order);
-      const numberOfQuestions = ref(activeSection.value.question_count);
-      const descriptionText = ref(activeSection.value.description);
-      const sectionTitle = ref(activeSection.value.section_title);
-
-      const originalFormData = {
-        selectedQuestionOrder: activeSection.value.learners_see_fixed_order,
-        numberOfQuestions: activeSection.value.question_count,
-        descriptionText: activeSection.value.description,
-        sectionTitle: activeSection.value.section_title,
-      };
+      const learners_see_fixed_order = ref(activeSection.value.learners_see_fixed_order);
+      const question_count = ref(activeSection.value.question_count);
+      const description = ref(activeSection.value.description);
+      const section_title = ref(activeSection.value.section_title);
 
       const formDataHasChanged = computed(() => {
         return !isEqual(
           {
-            selectedQuestionOrder: selectedQuestionOrder.value,
-            numberOfQuestions: numberOfQuestions.value,
-            descriptionText: descriptionText.value,
-            sectionTitle: sectionTitle.value,
+            learners_see_fixed_order: learners_see_fixed_order.value,
+            question_count: question_count.value,
+            description: description.value,
+            section_title: section_title.value,
```

Comment:
Renaming these to match the data saved in the quiz data made it easy to compare the values by way of lodash/pick below

Reply:
Might be good to add this as a code comment? Good to clarify that snake_case is being used rather than camelCase to match API data.